packages/ubuntu_service/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.3.2
```

Comment:
Seems like the script doesn't pick up "chore" commits

Reply:
Yeah, that's intentional, I just forgot to add a line in the manual versioning step